resources/l10n/uk_UA/LC_MESSAGES/loot.po

```diff
@@ -2,7 +2,7 @@ msgid ""
 msgstr ""
 "Project-Id-Version: LOOT 0.16.1\n"
```

Comment:
Please update the project version, see point 5 here:
https://github.com/loot/loot/blob/master/CONTRIBUTING.md#translating-the-loot-application

Reply:
Done